website/content/docs/enterprise/admin-partitions.mdx

```diff
@@ -7,9 +7,9 @@ description: Consul Enterprise enables you to create partitions that can be admi
 # Consul Enterprise Admin Partitions
 
 <EnterpriseAlert>
-  This feature requires{' '}
-  <a href="https://www.hashicorp.com/products/consul/">Consul Enterprise</a>{' '}
-  with the Governance and Policy module.
+  This feature is available in versions 1.11.0+ of
+  HashiCorp Cloud Platform (HCP) and self-managed Consul according to the{' '}
+  <a href="/docs/enterprise/feature-availability">enterprise feature matrix</a>.
```

Comment:
Ah, meaning you're recommending moving this content to a Requirements section and removing the "notice box" from the top? I suppose that does make more sense ... people probably want to know what the thing is first and whether it's valuable / useful to them, not what version they need :)

Do you think it would still be useful to give them a link here to the matrix of all features for Consul Enterprise (and its different forms)?